package.json

```diff
@@ -71,7 +71,7 @@
     "core-js": "^3.1.4",
     "cpx": "^1.5.0",
     "cross-env": "^5.2.0",
-    "cypress": "^3.2.0",
+    "cypress": "3.3.0",
```

Comment:
good approach is to use 3.2.0 cypress version because 3.3.0 causes failing tests

Reply:
I had to switch this to 3.3.0 because there was an issue in 3.2.0 that prevented this. Specifically https://github.com/cypress-io/cypress/issues/3742

Reply:
If i switch back to 3.2.0 then the test suite doesn't run because it doesn't boot.